studio/src/store.js

```diff
@@ -209,11 +210,47 @@ export function unlinkStoreFromHash(store) {
     store.removeMeta('hashLink');
 }
 
-// When the query param is populated or changes, switch to the 'content' page.
 Store.search.subscribe((value, oldValue) => {
     if (
         (!oldValue.query && value.query) ||
         (Boolean(value.query) && value.query !== oldValue.query)
     )
         Store.page.set('content');
 });
+
+document.addEventListener('DOMContentLoaded', () => {
```

Comment:
store.js is growing too fast.
can we move this in a separate file?
name suggestions: router.js, deeplink.js.

Reply:
I used your suggestion `router.js` and moved navigation-related stuff there.